tools/scripts/vsc_openocd_debug.mk

```diff
@@ -0,0 +1,30 @@
+#
+#	Makefile to create project dependent configs files for debugging with openocd
+#
+
+# Definitions
+SETTINGS_TEMPLATE	= $(NO-OS)/tools/scripts/templates/template_settings.json
+LAUNCH_TEMPLATE		= $(NO-OS)/tools/scripts/templates/template_launch.json
+SETTINGSJSON 		= $(VSCODE_CFG_DIR)/settings.json
+LAUNCHJSON 			= $(VSCODE_CFG_DIR)/launch.json
+
+# Determine gdb path
+GDB_BIN				= $(shell which gdb)
+# Determine openocd path - we are overwriting the stm32cube openocd
```

Comment:
i guess you want to remove these commented out lines from here

Reply:
Sorry for the confusion. I should have squashed the commits to one. This part is outdated. I will squash and resubmit.